pom.xml

```diff
@@ -74,7 +74,7 @@
         <dependency>
             <groupId>com.zepben.protobuf</groupId>
             <artifactId>evolve-grpc</artifactId>
-            <version>0.31.0-SNAPSHOT6</version>
+            <version>0.31.0-SNAPSHOT9</version>
```

Comment:
todo: This pulls in more than just your change, please fix the change log.

Reply:
Which I though would be covered by the mentioning of switchstate as 7 and 8 seems like implementation and fix.

Reply:
Yeah sorry @Esaotuelr, you are right. I totally misread that